contrib/capnproto/rb_data.capnp

```diff
@@ -129,6 +129,9 @@ struct RBEIMEvaluationReal @0xf8121d2237427a80 {
   interpolationJxWAllQp       @19 :List(List(Real));
   interpolationPhiValuesAllQp @20 :List(List(List(Real)));
   eimErrorIndicatorInterpData @21 :List(Real);
+  interpolationQruleOrder     @22 :List(Integer);
+  interpolationDxyzDxiElem    @23 :List(Point3D);
+  interpolationDxyzDetaElem   @24 :List(Point3D);
```

Comment:
I would suggest to include Dzeta here as well, at least as a placeholder. Since if we need to add it in the future it would be better to have it grouped with the Dxi and Deta values. It would be annoying if we added some other properties in the meantime and then the Dzeta would not be grouped with the others.

Reply:
I added interpolationDxyzDzetaElem to the schema as a placeholder but it's worth noting that it is not used in the code at the moment.